packages/flex-dev-utils/src/prints.ts

```diff
@@ -17,3 +17,5 @@ export const printList = (...lines: string[]) => {
   logger.info(multilineString(...lines));
   logger.newline();
 };
+
+export default {};
```

Comment:
i still don't like being able to import this without ts complaining (bc you wouldn't be able to use it) but since we'll remove unnecessary imports in v4, 👌for now